Notepad/HomeViewController.swift

```diff
@@ -17,20 +17,32 @@ class HomeViewController: UIViewController {
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
```

Comment:
Random question, do you know what the `sender` parameter is?

Reply:
no

Reply:
Find out

Reply:
it determines what triggered the action and using sender = any ensures flexibility between other UI elements

Reply:
Good shit